docs/cookbook/dump-variables.md

```diff
@@ -0,0 +1,14 @@
+# Dump variables
+Shopsys Framework uses Symfony with its VarDumper. 
+In addition to the standard function `dump()`, we implement the function `d()`, which you can pass any number of attributes where each one is dumped.
+
+Dumped variables are primarily available in the profiler. 
+
+Since we do not want to dump variables into the JSON response when developing the frontend API, the dumps are directed to the Symfony dump server. (https://symfony.com/doc/current/components/var_dumper.html#the-dump-server)
+
+You can find the debug settings in the file `config/packages/dev/debug.yaml`:
+
+Start the server with the `server:dump` command and whenever you call the `d()`, the dumped data will not be displayed in the output but sent to that server, which outputs it to its own console or to an HTML file.
```

Comment:
So do I understand it correctly that I have to run the command every time I want the dumps in the server console instead of the output? Or with the new settings, I do not need to do anything and I can always find the dumps in the profiler and they will not mess up the API response anymore out of the box?

Reply:
No, you don't. Dumps are still in profiller. Only mess from API responses are moved into `server:dump` :)